src/profile/client.builder.js

```diff
@@ -0,0 +1,66 @@
+
+const SandboxProfileClient = require('./client');
+const { Validation } = require('../util');
+const fs = require('fs');
+
+/**
+ * @class SandboxProfileClientBuilder
+ */
+class SandboxProfileClientBuilder {
+  /**
+   * @param {string} sdkId
+   */
+  withClientSdkId(sdkId) {
+    this.sdkId = sdkId;
+    return this;
+  }
+
+  /**
+   * @param {string} sdkId
+   *
+   * @deprecated 4.0.0
+   */
+  forApplication(sdkId) {
+    return this.withClientSdkId(sdkId);
+  }
+
```

Comment:
We can just remove this, right? No need to release as deprecated

Reply:
Removed in 919a0298c7f1b7a81c8a2d1cc69d0cbeec854759